auth-web/src/store/modules/business.ts

```diff
@@ -72,6 +72,34 @@ export default class BusinessModule extends VuexModule {
       })
   }
 
+  @Action({ rawError: true })
+  public async createNumberedBusiness (accountId: number) {
+    const requestBody: NumberedBusinessRequest = {
+      filing: {
+        header: {
+          name: FilingTypes.INCORPORATION_APPLICATION,
+          accountId: accountId
+        }
+      }
+    }
+
+    await BusinessService.createNumberedBusiness(requestBody)
+      .then(response => {
+        if (response && response.data && (response.status === 200 || response.status === 201)) {
```

Comment:
You could import the Http Codes npm package and then use a constant instead of 200. Up to you.

Reply:
I could for sure, good idea!

Reply:
After a little look through, this would involve a refactor of all the modules to remain consistent. So i'll note this change down, and implement it separately from this PR, just to keep things simple with our first steps here :)

Reply:
you could leave this to us @cameron-freshworks ... We will take care of it if you dont mind.. such a big change will need some regression effort as well..

Reply:
Absolutely, no problem there at all :) Thanks!